app/Services/InvoiceService.php

```
@@ -3,104 +3,93 @@
+        return $this->invoiceModel->shareInvoice($id);
     }
 
+    /**
+     * Get invoice PDF.
+     *
+     * @param string $id   Invoice ID
+     * @param string $type Type of PDF (stream, download, attach)
+     *
+     * @return mixed PDF output based on type
+     */
     public function getInvoicePdf($id, $type = 'stream')
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)